Dockerfile

```diff
@@ -11,6 +11,14 @@ COPY . /opt/kubemarine/
 WORKDIR /opt/kubemarine/
 
 RUN apt update && \
+    # Install Golang and build ipip_check
+    apt install -y wget  gcc && \
+    wget https://golang.org/dl/go1.19.8.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf go1.19.8.linux-amd64.tar.gz && \
+    /usr/local/go/bin/go mod download && \
+    GOOS=linux CGO_ENABLED=1 /usr/local/go/bin/go build -ldflags="-linkmode external -extldflags='-static'" -o kubemarine/resources/scripts/ipip_check -buildvcs=false kubemarine/resources/scripts/source/ipip_check/ipip_check.go && \
+    rm -Rf /usr/local/go go1.19.8.linux-amd64.tar.gz && \
+    apt autoremove -y gcc wget && \
```

Comment:
Is it possible to use multi-stage docker build, like https://docs.docker.com/language/golang/build-images/#multi-stage-builds

Reply:
Good idea, let's try